modules/builds-source-secrets-entitlements.adoc

```diff
@@ -10,20 +10,32 @@ Builds that use Red Hat subscriptions to install content must include the entitl
 
 .Prerequisites
 
-You must have access to Red Hat entitlements through your subscription. The entitlement secret is automatically created by the Insights Operator.
+You must have access to {op-system-base-full} package repositories through your subscription.
 
+The entitlement secret to access these repositories is automatically created by the Insights Operator when your cluster is subscribed.
 
-[TIP]
+[IMPORTANT]
 ====
-When you perform an Entitlement Build using {op-system-base-full} 7, you must have the following instructions in your Dockerfile before you run any `yum` commands:
+You must be a cluster administrator or have permission to access secrets in the `openshift-config-managed` project.
+====
+
+.Procedure
 
+. Copy the entitlement secret from the `openshift-config-managed` namespace to the build's namespace by using the following YAML content:
```

Comment:
Technically this isn't YAML content - the thing that _looks like YAML_ is a template that is fed into the `oc get secret` command. Not 100% sure how this should be phrased to customers (maybe "run the following commands")